src/inputs/Autocomplete.stories.tsx

```diff
@@ -16,7 +16,7 @@ Example.parameters = { chromatic: { disableSnapshot: true } };
 export const MenuOpen: StoryFn = Template.bind({});
 MenuOpen.play = async ({ canvasElement }: { canvasElement: HTMLElement }) => {
   const canvas = within(canvasElement);
-  await fireEvent.focus(canvas.getByTestId("heroes"));
+  canvas.getByTestId("heroes").focus();
```

Comment:
Ahh! Here's one where we went the other way! Granted, this is in Storybook....

Reply:
Yep, I needed "the real focus" here.

Fwiw not sure what the old code was doing b/c `fireEvent.focus` is not a promise.